tests/functional/test_read_configuration.py

```diff
@@ -55,3 +58,30 @@ def test_read_config(config_directory, config_file, expected):
     result = api.read_configuration(config_filename)
 
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "default_encoding",
+    (
+        "cp1252",
+        "UTF-8",
+    ),
+)
+def test_read_config_unicode(default_encoding, monkeypatch):
```

Comment:
Maybe it would be better to drop the parametrization (that's more suited to unit tests) and call it something like `test_read_utf8_config_on_windows`. What do you think?

Reply:
Yeah good point, I'll change that.